Cdn_GeneralPage_View.php

```diff
@@ -9,7 +9,7 @@
 		'key' => 'cdn.configuration_overloaded'
 	) );
 ?>
-<p><?php w3tc_e( 'cdn.general.header', 'Host static files with your <acronym title="Content Delivery Network">CDN</acronym> to reduce page load time.' ); ?>
+<p><?php w3tc_e( 'cdn.general.header', 'Host static files with your <acronym title="Content Delivery Network">CDN</acronym> to reduce page load time.', 'w3-total-cache' ); ?>
```

Comment:
`w3tc_e` does not have a 3rd parameter.  You added a text domain.

Reply:
Done!